Sources/Logto/Utilities/Utilities.swift

```diff
@@ -78,4 +78,28 @@ enum Utilities {
             Utilities.handleResponse(data: data, error: error, completion: completion)
         }
     }
+
+    static func httpPost(
```

Comment:
Do we need a test for this util method? i.e. test the the `unableToConstructUrl` exception ?

Reply:
absolutely, will do later. added LOG-1391